crates/db_views_actor/src/community_view.rs

```diff
@@ -194,6 +194,19 @@ impl CommunityView {
 
     PersonView::is_admin(pool, person_id).await
   }
+
+  pub async fn is_mod_of_any_or_admin(
```

Comment:
This name is confusign and easy to misread. I prefer to keep it as `is_community_mod_or_admin()`, with a comment to clarify that it checks for any community.

Reply:
The function right above this is called `is_mod_or_admin`, so that would be a duplicate name. I can add a comment tho.